src/plots/ternary/ternary.js

```diff
@@ -526,7 +526,7 @@ proto.initInteractions = function() {
                 dragOptions.clickFn = clickZoomPan;
                 dragOptions.doneFn = dragDone;
                 panPrep();
-                clearSelect(zoomContainer);
```

Comment:
Could we rename this thing to `zoomlayer` here? https://github.com/plotly/plotly.js/blob/8fea15620773d35f58c3ee6d6db00898e75d514f/src/plots/ternary/ternary.js#L496

Reply:
Ok done in -> 51b74b56d